sorbet.natvis

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+  <Type Name="sorbet::core::Name">
+    <DisplayString Condition="kind == 1">{utf8}</DisplayString>
+    <DisplayString>Name kind={kind}</DisplayString>
+  </Type>
+  <Type Name="sorbet::core::NameRef">
+    <!-- Keep bound check in sync with NAME_LAST_WELL_KNOWN_NAME from autogenerated file Names_gen.h -->
```

Comment:
We add entries to `generate_names.cc` all the time, is this `452` going to be super brittle?

Reply:
Definitely will be however as long as it doesn't shrink the only effect will be that less `NameRef` are recognized out of the box. Could also have that value set as another global in debug mode so that it can lookup based on that.